src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -33,7 +35,7 @@
  *
  * @author Kevin Bond <kevinbond@gmail.com>
  */
-class SymfonyStyle extends OutputStyle
+final class SymfonyStyle extends OutputStyle
```

Comment:
This is a BC break, but if we decide to mark it as final we can do this using @final annotation and make it final in Sf 6

Reply:
thx, I've removed the "final" keyword